src/components/LiquidityMining/Create/Steps/PreviewAndCreate/index.tsx

```diff
@@ -23,16 +40,54 @@ const ResponsiveContainer = styled(Box)<{ flex1?: boolean }>`
     margin-top: 16px !important;
```

Comment:
Shouldn't use `!important` here. If we need to override styles import from `rebass/styled-components` instead

Reply:
I blame luzzi xD